api/nvidia/v1/clusterpolicy_types.go

```diff
@@ -841,6 +843,60 @@ type SandboxDevicePluginSpec struct {
 	Env []EnvVar `json:"env,omitempty"`
 }
 
+// IMEXDRADriverSpec defines the properties for the NVIDIA IMEX DRA Driver deployment
+// TODO: add 'controller' and 'kubeletPlugin' structs to allow for per-component configuration
+type IMEXDRADriverSpec struct {
```

Comment:
If the DRA driver will eventually manage other resources, not just IMEX channels, it will be difficult to move away from the IMEX-specific settings for a DRA image - because of backward compatibility. If it's something that's going to evolve (I just don't know), I'd probably have a general DRA config (controller image, etc.), and under that a section that would control what kinds of resources and how the DRA driver could manage. IMEX channels in this case.